packages/resource-scheduler/src/components/ResourceScheduler/ResourceScheduler.scss

```diff
@@ -0,0 +1,14 @@
+.search-panel-wrapper {
+  height: 100%;
+  overflow: auto;
+  padding: 8px;
+}
+
+.Resizer.vertical {
+  background-color: transparent;
+  background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAUAAAAeCAYAAADkftS9AAAAIklEQVQoU2M4c+bMfxAGAgYYmwGrIIiDjrELjpo5aiZeMwF+yNnOs5KSvgAAAABJRU5ErkJggg==');
```

Comment:
Why background-image?

Reply:
its used to show the dot-image to indicate something resizeable